test/gtest/ucs/test_memtrack.cc

```diff
@@ -263,3 +263,46 @@ UCS_TEST_F(test_memtrack, custom) {
 
     test_total(1, ALLOC_SIZE);
 }
+
+
+class test_memtrack_log : public ucs::test {
+public:
+    static ucs_log_func_rc_t
+    zero_size_log_handler(const char *file, unsigned line, const char *function,
+                          ucs_log_level_t level,
+                          const ucs_log_component_config_t *comp_conf,
+                          const char *message, va_list ap)
+    {
+        std::string log_str = format_message(message, ap);
+        if ((level == UCS_LOG_LEVEL_WARN) &&
+            (log_str.find("allocated zero-size block") != std::string::npos)) {
+            ++log_count;
```

Comment:
maybe print the message using UCS_TEST_MESSAGE?
it's done in wrap_errors_logger for example